reflex/reflex.py

```diff
@@ -103,7 +101,7 @@ def _init(
     prerequisites.initialize_gitignore()
 
     # Initialize the requirements.txt.
-    prerequisites.initialize_requirements_txt()
+    dependency.initialize_requirements_txt()
```

Comment:
Should this be part of this package since it's a bit unrelated to the hosting? The initial one will only have the reflex package in it anyways. If we want to update this it may make it harder

Reply:
Synced on discord. The desired code is in here now: https://discord.com/channels/@me/1141437177448906882/1179140841089007700, so we good on this.

Reply:
updated in the latest rev